server/database/queries/appointments/getAppointmentsByPatientName.js

```diff
@@ -0,0 +1,12 @@
+const connection = require('../../config/connection');
+
+const getAppointmentsByPatientName = ({ firstName, lastName, phone }) => {
+  const sql = {
+    text:
+      'SELECT appointments.id,appointments.appointment_date,appointments.appointment_time,patients.firstname,patients.lastname FROM appointments INNER JOIN patients ON patients.id = appointments.patient_id where appointments.is_done =false AND patients.firstname = $1 AND patients.lastname = $2 OR (patients.phone= $3 AND appointments.is_done =false) ;',
```

Comment:
Please select the patient phone also, Assume the patient time come and he didn't arrive and the doctor wants to call him  then He want the phone number :)